dell/xps/15-9570/default.nix

```diff
@@ -0,0 +1,12 @@
+{ lib, ... }:
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/cpu/intel/kaby-lake
```

Comment:
Any particular reason `kaby-lake` is imported here, but not in `intel/` or `nvidia/`?

Reply:
Because here, `intel/` is the configuration for intel only and `nvidia/` means intel+nvidia
I could have imported kaby-lake in both files but as it is a common configuration, import it one time in the common file